python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -335,7 +335,9 @@ def get_materialized_partitions_subset(
     if not partitions_def:
         return None
 
-    if instance.can_cache_asset_status_data() and partitions_def in CACHEABLE_PARTITION_TYPES:
+    if instance.can_cache_asset_status_data() and any(
```

Comment:
Another bug I discovered along the way: during the `partitions_def in CACHEABLE_PARTITION_TYPES` check, a daily partitions definition != time window partitions definition. This PR fixes this by using an `isinstance` check instead.

Reply:
nit: if CACHEABLE_PARTITIONS_TYPE is a tuple, then isinstance(partitions_def, CACHEABLE_PARTITION_TYPES) will do the same thing.

Reply:
ooo this is nice, thanks!